Core/Config.cpp

```diff
@@ -1,4 +1,4 @@
-// Copyright (c) 2012- PPSSPP Project.
+﻿// Copyright (c) 2012- PPSSPP Project.
```

Comment:
Hmm trying to run EditorConfigPlugin.vsix it just tells me it found VS2017, but didn't found any product it can be installed to:P. I think I'll just manually edit that via some simpler editor.